src/bootchooser.c

```diff
@@ -152,16 +155,16 @@ static gboolean barebox_set_primary(RaucSlot *slot) {
 			continue;
 
 		if (s == slot) {
-			prio = prio1;
+			prio = BAREBOX_STATE_NEW_PRIORITY;
 		} else {
-			prio = prio2;
+			prio = BAREBOX_STATE_DEFAULT_PRIORITY;
 		}
 		g_ptr_array_add(pairs, g_strdup_printf("%s.%s.priority=%i",
 				BOOTSTATE_PREFIX, s->bootname, prio));
 	}
 
 	g_ptr_array_add(pairs, g_strdup_printf("%s.%s.remaining_attempts=%i",
-			BOOTSTATE_PREFIX, slot->bootname, 3));
+			BOOTSTATE_PREFIX, slot->bootname, BAREBOX_STATE_NEW_ATTEMPS));
```

Comment:
This should be `BAREBOX_STATE_DEFAULT_ATTEMPS`, too. There is no difference in meaning here.